### DISCUSSION TITLE
Missing objects

### DISCUSSION THREAD

**Discussion by aeszter** [streetcomplete/StreetComplete#4304] [Q&A] [answered]
Are there any recommendations on how to deal with (possibly) missing objects?
When I'm on a survey and encounter an object that I suspect to be missing from the map, how do I check to be sure? How do I go about adding it? Can StreetComplete help here, or can you recommend any other (mobile-friendly) app for this task?

Thanks,

A.

**Comment by matkoniecz** [comment, marked as answer]
For now I recommend Vespucci. StreetComplete may show no quests in a given area as POIs are missing or because all were answered.

For objects always shown on map (buildings/roads/rivers and so on) you can make a note with a photo and GPX track but editing directly with Vespucci is even better.

**Reply by FloEdelmann** [reply]
Other mobile editors (alternatives to [Vespucci](https://wiki.openstreetmap.org/wiki/Vespucci)): [Osm Go!](https://wiki.openstreetmap.org/wiki/Osm_Go!) and [Every Door](https://wiki.openstreetmap.org/wiki/Every_Door)

**Reply by aeszter** [reply]
Thanks, I'll give them a try,

**Reply by aeszter** [reply]
For completeness' sake: OsmAnd can also add point features/nodes.

**Comment by westnordost** [comment]
At this point, I pretty much recommend Every Door for this **unless** you want to modify geometry (draw roads, buildings,...). But doing that on a mobile device is something I would not recommend anyway.

**Reply by matkoniecz** [reply]
In my experience road drawing or fixing broken road geometries is much easier in Vespucci than making notes and then discovering that it is not enough to map in JOSM.
